extensions/wikia/ThemeDesigner/ThemeDesignerHooks.class.php

```diff
@@ -61,12 +55,14 @@ public static function onUploadComplete( $image ) {
 	 * @return bool return false to prevent the upload
 	 */
 	public static function onUploadVerification( $destName, $tempPath, &$error ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$tempPath". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)